packages/components/src/plugins/index.js

```diff
@@ -54,11 +50,11 @@ export class Plugins extends Component {
 
 		installAndActivatePlugins( pluginSlugs )
 			.then( ( response ) => {
-				createNoticesFromResponse( response );
+				onResponse( response );
```

Comment:
I'm a little wary of introducing a new prop for the response when we have `onComplete` and `onError`.  What do you think about passing the full response to `onComplete` and `onError` so we can handle this without introducing a new prop?